postgresql/cluster/init.sls

```diff
@@ -0,0 +1,14 @@
+{%- from "postgresql/map.jinja" import cluster with context %}
+
+{%- if cluster.enabled %}
+include:
+{%- if cluster.get("mode") == "hot_standby" and cluster.get("role") == "master" %}
+- postgresql.cluster.hot-standby.master
```

Comment:
I don't know if underscore isn't better in sls names (check convention to be sure)